### PR TITLE
CLI updates

### PR DESCRIPTION
**Pull request by jlpoltrack**
@olliw42 
Some updates to CLI, could you review the 'stats' section - I based the names from the CLI code but if there are better descriptions - let me know.

### CODE REVIEW

File: docs/CLI.md [hunk 1]
```diff
@@ -48,9 +48,15 @@ Starts the binding. Comment: Only the Tx module is set into binding mode. The re
 Reloads the current parameter values from the Tx and, if connected, the receiver.
 
 #### stats; #### 
-Starts streaming some statistics. Terminate by sending any character.
+Starts streaming statistics. Terminate by sending any character. Format:
+
+<table>
+    <tr>
+        <td>Tx link quality</td><td>Rx valid frames link quality</td><td>Rx link quality</td><td>Rx last RSSI</td>
```
Comment (olliw42) [inline docs/CLI.md:55]: Tx LQ | I need to think about it | Rx LQ | Tx RSSI | Rx RSSI | Tx SNR | Tx bytes/sec transmitted | Tx bytes/sec received

I'm not sure from top of my head about valid_frames_received.GetLQ(), I would think it's the same as before

Comment (olliw42) [reply docs/CLI.md:55]: maybe for the moment you could write "Tx valid frames LQ"

Comment (jlpoltrack) [reply docs/CLI.md]: Sounds good - updated.

**Comment by olliw42** [comment]
good changes 😀
LGTM

**Pull request merged**